vms/platformvm/state/state.go

```diff
@@ -1285,7 +1285,8 @@ func (s *state) syncGenesis(genesisBlk block.Block, genesis *genesis.State) erro
 
 	// Persist UTXOs that exist at genesis
 	for _, utxo := range genesis.UTXOs {
-		s.AddUTXO(utxo)
+		avaxUtxo := utxo.UTXO
+		s.AddUTXO(&avaxUtxo)
```

Comment:
I think we typically capitalize the whole abbreviation
```suggestion
		avaxUTXO := utxo.UTXO
		s.AddUTXO(&avaxUTXO)
```